tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -19,81 +21,158 @@
+							name: 'wp-debug-log',
+						},
+						{
+							label: 'Multisite based on subdomains',
+							name: 'subdomain_multisite',
+						},
+						{
+							label: 'Multisite based on subdirectories',
+							name: 'subdir_multisite',
+						},
+						{
+							label: 'Pre-generate content',
+							name: 'content',
+						},
+						{
+							label: '<code>xmlrpc.php</code> unavailable',
+							name: 'blockxmlrpc',
+						},
+					],
+					100
+				) }
+				<h4>Plugins</h4>
+				${ getOptionsList(
+					[
+						{
+							label: 'WordPress Beta Tester',
+							name: 'wordpress-beta-tester',
+						},
+						{
+							label: 'Gutenberg',
+							name: 'gutenberg',
+						},
+						{
+							label: 'Classic Editor',
+							name: 'classic-editor',
+						},
+						{
+							label: 'WooCommerce',
+							name: 'woocommerce',
+						},
+						{
+							label: 'WooCommerce Beta Tester',
+							name: 'woocommerce-beta-tester',
+						},
+						{
+							label: 'Config Constants',
+							name: 'config-constants',
+						},
+						{
+							label: 'Code Snippets',
+							name: 'code-snippets',
+						},
+						{
+							label: 'WP Rollback',
+							name: 'wp-rollback',
+						},
+						{
+							label: 'WP Downgrade',
+							name: 'wp-downgrade',
+						},
+						{
+							label: 'WP Super Cache',
+							name: 'wp-super-cache',
+						},
+						{
+							label: 'WP Log Viewer',
+							name: 'wp-log-viewer',
+						},
+						{
+							label: 'WP Job Manager',
+							name: 'wp-job-manager',
+						},
+					],
+					33
+				) }
+				<p>
+					<a id="jetpack-beta-branch-link" target="_blank" rel="nofollow noopener" href="${ link }">
+						${ link }
+					</a>
+				</p>
+			`;
+			appendHtml( markdownBody, contents );
 		} else if ( ! branchIsMerged ) {
-			appendHtml( markdownBody, branchIsForkedText );
+			appendHtml(
+				markdownBody,
+				"<p><strong>This branch can't be tested live because it comes from a forked version of this repo.</strong></p>"
+			);
 		} else {
-			appendHtml( markdownBody, branchIsMergedText );
+			appendHtml( markdownBody, '<p><strong>This branch is already merged.</strong></p>' );
+		}
+
+		function getLink() {
+			const query = [ 'jetpack-beta', `branch=${ branch }` ];
+			$( '#jetpack-live-branches input[type=checkbox]:checked' ).each( ( i, input ) => {
+				query.push( input.name );
+			} );
+			return `https://jurassic.ninja/create?${ query.join( '&' ) }`;
+		}
+
+		function getOption( { checked = false, label, name }, columnWidth ) {
+			return `
+			<li style="min-width: ${ columnWidth }%">
+				<label style="font-weight: inherit; ">
+					<input type="checkbox" name="${ name }" ${ checked ? 'checked' : '' }>
+					${ label }
+				</label>
+			</li>
+			`;
 		}
 
-		function appendHtml( el, str ) {
-			const div = document.createElement( 'div' );
+		function getOptionsList( options, columnWidth ) {
+			return `
+				<ul style="list-style: none; padding-left: 0; display: flex; flex-wrap: wrap;">
```

Comment:
Inline CSS instead of relying on Github, because relying on those classes was causing occasional JS fatal errors.